core/drand_beacon.go

```diff
@@ -434,6 +447,98 @@ func (bp *BeaconProcess) newMetadata() *common.Metadata {
 	return metadata
 }
 
+var errNoRoundInPeers = errors.New("could not find round")
+
+func (bp *BeaconProcess) storeCurrentFromPeerNetwork(ctx context.Context, store chain.Store) error {
+	clkNow := bp.opts.clock.Now().Unix()
+	currentRound := chain.CurrentRound(clkNow, bp.group.Period, bp.group.GenesisTime)
+	if currentRound < 1 {
+		// We cannot sync the initial round.
+		// Assume this is a fresh start
+		return nil
```

Comment:
If this is a fresh start, I'm not sure you can do `bp.group.GenesisTime` without it panicking. Can you? 
Have you tested creating a new network with one of the nodes using the memdb backend?